src/filters/type.js

```diff
@@ -75,6 +77,37 @@ params:
 const mdInline = (content) =>
   content ? typogr(mdown.renderInline(content), true) : content;
 
+/* @docs
+label: elide
+category: typesetting
+note: |
+  Elide HTML at a given word count,
+  and append `…` if elided.
+params:
+  html:
+    type: string
+  count:
+    type: Number
+    default: 50
+*/
+const elide = (html, count = 50) => {
+  // Strip links, paragraph breaks, etc:
+  const stripped = striptags(html.trim(), ['code', 'strong', 'em']);
+  // Truncate stripped html:
+  let truncated = truncate(stripped, count, { byWords: true, ellipsis: '…' });
+  // This will catch both cases where no truncation was needed, but also
+  // (unintentionally) cases where the truncation is within a final tag
+  // (e.g. `<em>final words…</em>`) -- but that's probably good enough for now?
+  if (!truncated.endsWith('…')) {
+    return truncated;
+  }
+  // Strip non-alphanumeric trailing chars (e.g. commas, periods):
+  if (truncated.slice(-2, -1).match(/[^A-Z|a-z|0-9]/) !== null) {
+    truncated = `${truncated.slice(0, -2)}…`;
+  }
+  return truncated;
+};
```

Comment:
@mirisuzanne This is somewhere in between what we had before, and an ideal solution. 
Unfortunately the `truncate-html` library doesn't handle e.g. `"ends in a comma,…"` very well, so I left in your workaround for that, which covers _most_ cases (see comments above about remaining edge cases, of which I can't actually find any currently in practice).

Reply:
@jgerigmeyer looks like a step up from what I had. Merge it?